NEWS.rst

```diff
@@ -53,15 +53,24 @@ property giving the numerical identifier for the REBASE database identifier
 from which the enzyme object was created, and a `uri` property with a canonical
 `identifiers.org` link to the database, for use in linked-data representations.
 
+The ``SeqUtils`` module has two new submodules: ``ctd`` and ``pseaac``, which
+provide new Classes for calculating the Composition Transition and
+Distribution descriptors and the Pseudo Amino Acid Composition of a protein
+respectively. New methods for k-mer counting have also been added to
+``ProtParam.ProteinAnalysis``. Both additions have the goal of making the
+process of generating numerical descriptors for proteins easier.
+
 Additionally, a number of small bugs and typos have been fixed with additions
 to the test suite.
 
 Many thanks to the Biopython developers and community for making this release
 possible, especially the following contributors:
 
+- Aziz Khan
```

Comment:
I'm guessing dropping this name was an accident via a merge?

Reply:
Yes, my bad